cross/build-rootfs.sh

```diff
@@ -117,6 +120,9 @@ done
 if [ "$__BuildArch" == "armel" ]; then
     __LLDB_Package="lldb-3.5-dev"
 fi
+if [ "$__BuildArch" == "arm64" ]; then
+    __LLDB_Package="lib$__LLDB_Package"
```

Comment:
I believe the lib thing is not arm64 specific, but rather lldb version specific. If I recall it correctly, it has started with llvm 3.9.

Reply:
Actually, it seems that there was always liblldb-3.x-dev package for all platforms and the lldb-3.x-dev was just a transitional forwarding package.
From 3.9 on, the old lldb-3.x-dev was dropped.

Reply:
(checked on the debian packages list site)

Reply:
Done